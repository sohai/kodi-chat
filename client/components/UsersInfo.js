import React from 'react';
import { pure, compose } from 'recompact';
import PropTypes from 'prop-types';

const propTypes = PropTypes && {};
const defaultProps = {};

export const UsersInfoComponent = ({ you, contact }) => (
  <div>
    You({you.name}) are talking with {contact.name || 'noone'}
  </div>
);

UsersInfoComponent.displayName = 'UsersInfo';
UsersInfoComponent.propTypes = propTypes;
UsersInfoComponent.defaultProps = defaultProps;
export default compose(pure)(UsersInfoComponent);
