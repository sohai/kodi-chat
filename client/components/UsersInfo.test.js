/* global render, describe, it, expect */
/* eslint-disable import/first */
import React from 'react';
import UsersInfo from './UsersInfo';

describe('(Component) UsersInfo', () => {
  it('should render', () => {
    const props = {
      you: {
        name: 'you'
      },
      contact: {
        name: 'name'
      }
    };
    const wrapper = render(<UsersInfo {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
