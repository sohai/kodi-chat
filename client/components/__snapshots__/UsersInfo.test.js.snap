// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) UsersInfo should render 1`] = `
<div>
  You(you) are talking with name
</div>
`;
